src/currencies.json

```diff
@@ -6,7 +6,7 @@
   "ALL": { "name":"Albanian Lek",                         "decimals":2, "crypto":false },
   "AMD": { "name":"Armenian Dram",                        "decimals":2, "crypto":false },
   "ANG": { "name":"Netherlands Antillean Guilder",        "decimals":2, "crypto":false },
-  "ANKR": { "name":"Ankr Network ERC-20",                 "decimals":18, "crypto":true },
+  "ANKR": { "name":"Ankr Network ERC-20",                 "decimals":8, "crypto":true },
```

Comment:
I assume this is safe to do and we don't have any data stored with 18 decimals

Reply:
but please confirm :D

Reply:
we are only just enablink ankr now, so i kind of assumed myself we didn't have any data on this. 
I have looked through swaps destination_currency and ankr is not there, are there any other places i need to look?

Reply:
should be ok